src/browser/core.js

```diff
@@ -45,15 +71,16 @@ define(function(require, exports, module) {
         }
       }
 
-      // create a cursor for a current up to date state with a
-      // step as change handler, to retriger render loop on change.
-      const cursor = Cursor.from(current, step);
-      // Finally use React to render current state with a component.
-      React.render(Component(cursor), target);
+      if (window.debug) {
+        console.log('! update', path.join('.'), current.toJSON());
+      }
+
+
+      schedule(draw);
     }
 
     // Spawn render loop by stepping into!
-    step(initial, null);
+    step(initial, null, []);
```

Comment:
> Should empty array be here?

It's so that that code that does logging would always have a path to print, without it's going to throw.